analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -52,7 +54,17 @@ class SpdxDocumentFileFunTest : StringSpec() {
             actualResult shouldBe expectedResult
         }
 
-        // TODO: Add a test for "package.spdx.yml".
+        "SPDX yaml files without a project package are removed from the definitionFiles" {
```

Comment:
Nit: I believe it's fine to not be specific to YAML here, and maybe just say "mapDefinitionFiles() removes SPDX documents that do not describe a project".